guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -72,7 +72,7 @@ ifdef::foreman-el,katello[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# yum localinstall https://fedorapeople.org/groups/katello/releases/yum/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+# yum localinstall https://yum.theforeman.org/katello/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
```

Comment:
Looking at the other file, this entire file should use `el{distribution-major-version}` instead of `el7`.

Reply:
probably correct, but it also talks about epel and scls, so needs more than just that for el8 awareness

Reply:
Yes, it needs a lot of love.